src/ResponseFactory.php

```diff
@@ -9,7 +9,6 @@ class ResponseFactory implements ResponseFactoryInterface
 {
     public function createResponse($code = 200)
     {
-        return (new Response())
-            ->withStatus($code);
+        return new Response('php://memory', $code);
```

Comment:
Having to define `$body` is why I didn't write it this way. What's the intended benefit of this change?